dnf/cli/commands/repoquery.py

```diff
@@ -90,7 +90,7 @@ class RepoQueryCommand(commands.Command):
     """A class containing methods needed by the cli to execute the repoquery command.
     """
 
-    aliases = ('repoquery',)
+    aliases = ('repoquery', 'deplist')
```

Comment:
@jsilhan @j-mracek Do we have a solution for this? Will it be implemented using yum wrapper?